File: Core/Entities/Grupa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Core.Entities
{
    public class Grupa
    {
        public int Id { get; set; }

        public ICollection<AppUser> Elevi { get; set; }

        public DateTime DataGrupa { get; set; }

        public DateTime OraGrupa { get; set; }

        public int LocatieId { get; set; }

        public Locatie Locatie { get; set; }

        public bool Confirmata { get; set; }

        public bool Efectuata { get; set; }

        public int NivelId { get; set; }
    }
}

File: client/src/app/models/grupa.ts
import { Elev } from "./elev";

export class Grupa {
    id: number;
    dataGrupa: Date;
    oraGrupa: Date;
    locatieId: number;
    elevi: Elev[] = [];
    particip: boolean;
    confirmata: boolean;
    efectuata: boolean;
    nivel: string;
    nivelId: number;
}
